app/models/manageiq/providers/redhat/infra_manager/vm/operations/snapshot.rb

```diff
@@ -55,12 +55,21 @@ def allowed_to_revert?
     current_state == 'off'
   end
 
+  def revert_to_snapshot_denied_message(active = false)
```

Comment:
where is the `active` param used? I only see the call to this method in the `supports` block above